aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -149,8 +150,21 @@ async def create_invitation(
         if multi_use:
             invitation_mode = ConnRecord.INVITATION_MODE_MULTI
 
+        if not recipient_keys:
```

Comment:
if not ... else is a sign of overthinking: reverse these

Reply:
I see it now. I will fix that.